cmd/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/olekukonko/tablewriter v0.0.1
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3 // indirect
-	golang.org/x/crypto v0.0.0-20190411191339-88737f569e3a
+	golang.org/x/crypto v0.0.0-20200602180216-279210d13fed
```

Comment:
are we really using crypto or is this a redundant import?

Reply:
We are using it for reading passwords from terminal. https://github.com/wso2/product-mi-tooling/blob/master/cmd/cmd/secretInit.go#L29 https://github.com/wso2/product-mi-tooling/blob/master/cmd/utils/utils.go#L36